src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -71,27 +136,135 @@ const ChatItemTargetText = ({
   time = '4h',
   isContinuous = false,
   message = '',
+  attachments = [],
   avatar
 }: ChatItemMyTextProps) => {
+  const navigation = useNavigation();
+
   const renderAvatar = React.useCallback(() => {
     if (isContinuous) return <View style={styles.avatar} />;
     return <View style={styles.mr8}>{avatar}</View>;
   }, []);
 
+  const handleTextContainerStyle = () => {
+    const isAttachment = attachments.length > 0;
+    const paddingStyle = {
+      paddingLeft: isAttachment ? BUBBLE_LEFT_PADDING_ATTACHMENT : BUBBLE_LEFT_PADDING,
+      paddingRight: isAttachment ? BUBBLE_RIGHT_PADDING_ATTACHMENT : BUBBLE_RIGHT_PADDING
+    };
+    return [styles.textContainer, paddingStyle];
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's great to see the use of hooks and memoization to optimize rendering. The `handleTextContainerStyle` function is a good way to dynamically adjust styles based on whether attachments are present or not.



<!-- This is an auto-generated comment by OSS CodeRabbit -->